src/main/java/lotto/LottoPrice.java

```diff
@@ -0,0 +1,9 @@
+package lotto;
+
+public class LottoPrice {
+    private final int price = 1000;
+
+    public int lottoTicketNumber(int receiveMoney) {
+        return receiveMoney / price;
+    }
```

Comment:
```suggestion
    public int getLottoTicketNumber(int receiveMoney) {
        return receiveMoney / price;
    }
```

메서드명의 시작은 항상 영어 동사가 될 수 있도록 부탁드립니다.

Reply:
저번 수업에서
엘레강트 오브젝트 관련하여
Builder메서드는 명사로 그 외에는 Manipulator로서 void반환에 동사로...
빌더는 void가 아닌 반환값이 있는 메서드를 말함
따라서 메서드에 의미를 강조하기위해서는 Builder에는 형용사, Manipulator에는 부사
라는 내용을 듣고, 그에 맞게 해봤는데, 말씀하신대로 할까요? 😭

Reply:
@cptbong

봉일균님 방식이 더 적절한 것 같습니다. 제 생각을 시정합니다.
"메서드명의 시작은 항상 영어 동사가 될 수 있도록 부탁드립니다." -> 이 피드백은 철회합니다. 기억에서 삭제해주세요 😅 

스프링 프레임워크를 사용하다보면, 자바빈을 활용하기 위해 너무도 당연하게 getter 메서드를 사용하다 보니,
엘레강트 오브젝트의 관점을 제가 잊고 있었습니다.
포비님의 영상과, 책을 다시 읽으며 잘못 생각했었음을 깨달았습니다.